app/assets/stylesheets/pregnancies.scss

```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
   }
 }
 
-#menu {
-  border-right: 1px solid black;
+#sections {
+  border-left: 1px solid black;
+  padding-left: 15%;
```

Comment:
Ah, I see what you mean - my mistake, I thought this was what was breaking grid! If the Patient Information header lines up with LMP at intake, I'm good with it.